src/Resources/Locales/en.json

```diff
@@ -1376,11 +1373,17 @@
                 "edit": "Edit time series",
                 "remove": "Remove time series",
                 "property": "Property",
-                "propertyTypeCastLabel": "Cast property type to number"
+                "propertyTypeCastLabel": "Cast property type to number",
+                "noData": "This time series contains no data to display"
```

Comment:
can we trim this? `This time series contains no data`

Reply:
it is changed to something else based on the content feedback by Azima and Riley